contracts/Registry.sol

```diff
@@ -0,0 +1,149 @@
+//SPDX-License-Identifier: Unlicensed
+pragma solidity ^0.7.3;
+
+import "./IRegistry.sol";
+import "./IDelegate.sol";
+import "hardhat/console.sol";
+
+contract Registry is IRegistry {
+    uint64 private idSequence = 1;
```

Comment:
I changed this to 1000, which should be plenty. I believe this will be slightly more expensive for early adopters, but shouldn't make a difference in the overall economics.